src/main/java/org/maxgamer/quickshop/shop/VirtualDisplayItem.java

```diff
@@ -390,13 +405,11 @@ public void sendFakeItem(@NotNull Player player) {
         return null;
     }
 
-    @Override
-    public @NotNull Location getDisplayLocation() {
-        return shop.getLocation().clone().add(0.5, 1.2, 0.5);
-    }
-
     @Override
     public boolean isSpawned() {
+        if (shop.isLeftShop()) {
+            return (Objects.requireNonNull(shop.getAttachedShop().getDisplayItem())).isSpawned();
+        }
```

Comment:
NullPointerException?

Reply:
If it's a left shop, it must have an attached shop not equal to null.